src/features/page_view_timing/aggregate/index.js

```diff
@@ -27,11 +27,7 @@ export class Aggregate extends AggregateBase {
     this.timings = []
     this.timingsSent = []
     this.curSessEndRecorded = false
-
-    try { // we (only) need to track cls state because it's attached to other timing events rather than reported on change...
-      this.clsSupported = PerformanceObserver.supportedEntryTypes.includes('layout-shift')
```

Comment:
Does web-vitals already do this check?

Reply:
Yep it does; only reports when its own compatibility is met